docs/docs/services/qix-engine/access-control.md

```diff
@@ -84,139 +143,147 @@ Example JWT:
 }
 ```
 
-Example usage in condition:
+Here, `user.id` will be set to `john-doe` based on the `sub` attribute. `employeeType`, and `custom.country`
+become attributes on `user`. The attributes could be used in a (fictitous) rule like this:
 
 ```c
-// Evaluates to `true` if the JWT looks like the example above:
-user.employeeType = "developer" and user.custom.country = "sweden"
+user.id = "john-doe" and user.employeeType = "developer" and user.custom.country = "sweden" and resource._actions = "*"
 ```
 
-### App
+With the example JWT above, this user is granted access for all actions to all resources.
 
-The document (called App in the rules syntax) context.
+### The `resource` Object
 
-It has no additional attributes.
+A _Resource_ is a generic concept that can represent documents, or objects within documents. A Resource carries attributes
```

Comment:
The engine interfaces (The REST interfaces for example) has the apps prefix. Should this documentation refer to apps or documents?

Reply:
Good point. It should be consistent. What about sticking to "apps"?

Reply:
I concur